lib/affirm/charge_event.rb

```diff
@@ -0,0 +1,20 @@
+module Affirm
+  class ChargeEvent
+    attr_reader :id, :transaction_id, :type, :created,
+      :amount, :fee, :order_id, :shipping_carrier, :shipping_confirmation,
+      :fee_refunded
+
+    def initialize(attrs)
```

Comment:
I feel like we'd be better off with something like [Hashr](https://github.com/svenfuchs/hashr) or a manual `method_missing` that checks against the hash's keys. This seems like a pain if they add a field to their response.